src/common/SConscript

```diff
@@ -11,8 +11,8 @@ def scons():
     # in the rpath.   It is normally only included by libpmemobj
     # but that library doesn't contain the installed location
     # in the rpath.
-    prereqs.require(env, 'pmdk', 'argobots', 'cart', 'readline',
-                    'protobufc', pmdk_libs=['pmemobj', 'pmem'])
+    prereqs.require(env, 'pmdk', 'cart', 'protobufc')
+    #pmdk_libs=['pmemobj', 'pmem'])
```

Comment:
So is the hack in the comment no longer needed?

Reply:
> So is the hack in the comment no longer needed?

Good question.  I couldn't work out what this specific code was doing, why is it passing an array assignment into a function anyway?

I was playing with removing pmemobj and pmem from the daos_client as well because I don't think they should be needed but obviously forgot to finish this.

Reply:
it overrides the default setting from the definition in scons_local.   In there, libpmemobj is the library configured.   This was replacing the list used.   It looks like a workaround for _something_ :-)